pvlib/tests/test_spectrum.py

```diff
@@ -276,36 +276,31 @@ def test_spectral_factor_firstsolar_ambiguous_both():
         spectrum.spectral_factor_firstsolar(1, 1, 'cdte', coefficients=coeffs)
 
 
-def test_spectral_factor_firstsolar_large_airmass():
-    # test that airmass > 10 is treated same as airmass==10
-    m_eq10 = spectrum.spectral_factor_firstsolar(1, 10, 'monosi')
-    m_gt10 = spectrum.spectral_factor_firstsolar(1, 15, 'monosi')
-    assert_allclose(m_eq10, m_gt10)
```

Comment:
@RDaxini the codecov check is failing because this rest was removed.  We should reinstate it, plus add another few lines that check the case where `max_airmass_absolute` is a specified value instead of left as the default.

Reply:
> the codecov check is failing because this test was removed.

I didn't realise that. I have reinstated it.

> add another few lines that check the case where `max_airmass_absolute` is a specified value instead of left as the default.

Good suggestion. I have added the following in the last commit:
```
def test_spectral_factor_firstsolar_large_airmass_supplied_max():
    # test airmass > user-defined maximum is treated same as airmass=maximum
    m_eq10 = spectrum.spectral_factor_firstsolar(1, 11, 'monosi',
                                                 max_airmass_absolute=11)
    m_gt10 = spectrum.spectral_factor_firstsolar(1, 15, 'monosi',
                                                 max_airmass_absolute=11)
    assert_allclose(m_eq10, m_gt10)
```
Let me know if that's okay
@kandersolar

Reply:
The logic looks right to me!  I would have named the variables `m_eq11` and `m_gt11` though :P

Reply:
Glitch in the matrix, restarted the job and all looks fine now